src/styles/_variables.js

```diff
@@ -1,4 +1,4 @@
 export const // Media Query Breakpoints
   $mobileBreakpoint = '578px',
   $tabletBreakpoint = '768px',
-  $lgDesktopBreakpoint = '1440px'
+  $lgDesktopBreakpoint = '1200px'
```

Comment:
i thought we decide 1280... but dont worry about this file. we neeed to rewrite.

Reply:
done